c2corg_api/tweens/rate_limiting.py

```diff
@@ -59,9 +59,12 @@ def tween(request):
                 if user.ratelimit_times > max_times:
                     user.blocked = True
 
-                # An alert message is sent to the moderators
-                email_service = get_email_service(request)
-                email_service.send_rate_limiting_alert(user)
+                try:
+                    # An alert message is sent to the moderators
+                    email_service = get_email_service(request)
+                    email_service.send_rate_limiting_alert(user)
+                except:
+                    log.debug('RATE LIMITING: error while sending email')
```

Comment:
Can you add exc_info=True, to get the trace in the log.
Are you sure the send mail timeout occurs before the http server timeout ?

Reply:
As I understand, the mail is sent when the transaction is commited (pyramid_tm tween) not immediately, except you use the appropriate method.
I do not have found timeout parameter or setting in the pyramid_mailer docs.

Reply:
Does this occurs on production, or only on your workstation because you cannot reach default config smtp server (psemail) ? Maybe we do not really need to handle this error.

Reply:
I only had this case on my workstation because indeed psemail cannot be used. It seems to work on the demo (not in prod yet). Maybe you're right and this change is not really useful, this can be removed.

Reply:
The prod server already send other emails using the same method for password recovery.
This try / except block has no effect on timeout that occurs in pyramid_tm tween.

Reply:
You're right. I have removed the commit adding the try/except block.

I will merge the PR when Travis has passed. Thanks for the feedback and advices.